src/utils/extensions/required-extensions.ts

```diff
@@ -0,0 +1,104 @@
+import BaseCommand from '../../commands/base-command.js'
+import { getAccount } from '../../commands/database/utils.js'
+import {
+  Extension,
+  ExtensionMeta,
+  getExtension,
+  //  getExtensionsMeta,
+  getInstalledExtensionsForSite,
+  installExtension,
+} from './utils.js'
+
+export async function installRequiredExtensions(command: BaseCommand) {
```

Comment:
I would do all the logic around getting and installing the extensions inside of this part in `@netlify/config`

https://github.com/netlify/build/blob/6b34ebf300dc45606cf787e7127b5b376b69e574/packages/config/src/api/site_info.ts#L39

Reply:
which could be the same logic!! Using the same feature flag :)

Reply:
Just make sure that everything still works when the feature flag is off 🙃